### DISCUSSION TITLE
Math generated with sphinx.ext.imgmath is not readable in dark mode

### DISCUSSION THREAD

**Discussion by tdegeus** [pradyunsg/furo#508] [Potential Issue] [answered]
Consider a very simple document:

```rst

Foo
===

.. math::

    a = b
```

configured as 

```python
project = "Foo"
copyright = "MIT"
author = "Bar"
extensions = ["sphinx.ext.imgmath"]
exclude_patterns = ["_build", "Thumbs.db", ".DS_Store"]
html_theme = "furo"
```

Then the white-background version is perfectly readable:
<img width="867" alt="bright" src="https://user-images.githubusercontent.com/7030445/186948229-1027827f-fdb1-4d4f-9787-618581fd1186.png">

but the equation is completely unreadable in dark mode
<img width="867" alt="dark" src="https://user-images.githubusercontent.com/7030445/186948304-1e59a429-7b53-4895-aa22-f1ae209e0460.png">

It would be great if there would be a way to invert the colours. Or at least render black as white in the equation.

**Comment by pradyunsg** [comment]
Without a screenshot, a hosted webpage that I can look at and/or the source code used to build that webpage, there's no way for me to figure out what's happening. :)

**Reply by tdegeus** [reply]
Sorry, you are completely right. Edited.

**Comment by pradyunsg** [comment, marked as answer]
Ah right! Using [sphinx.ext.mathjax](https://www.sphinx-doc.org/en/master/usage/extensions/math.html#module-sphinx.ext.mathjax) instead will work -- I don't think I've ever seen someone use [sphinx.ext.imgmath](https://www.sphinx-doc.org/en/master/usage/extensions/math.html#module-sphinx.ext.imgmath) "in the wild". 😅 

If someone really wants to use imgmath for some reason, lemme know and I'd be happy to explore implementing this -- I've never actually installed latex on my MacBook, to figure out how to get imgmath working correctly. :)

**Reply by tdegeus** [reply]
I did not really think about the difference. I guess it was some hit I got on the internet a million years ago that I'm still using... Thanks!
